collections/_posts/2021-09-01-public-bluebox-ami-release.md

```diff
@@ -0,0 +1,89 @@
+---
+lang: en
+layout: post
+title:  "Chia Releases Public AWS Image for Bluebox Timelords"
+date:   2021-09-01
+author: "[Patrick Maslana](https://keybase.io/pmaslana3)" , [Chris Marslender](https://keybase.io/pmaslana3), and [J. Eckert](https://twitter.com/sargonas)
+---
+
+#Chia Blockchain - Bluebox Timelord AMI Release
```

Comment:
GitHub wont render titles right without a space after the `#` - not sure about Jekyll